1. Andrews, S., 2010 FastQC: A quality control tool for high throughput sequence data. Babraham Bioinformatics
2. Antipov, D., A. Korobeynikov, J. S. McLean, and P. A. Pevzner, 2016 hybridSPAdes: an algorithm for hybrid assembly of short and long reads. Bioinformatics 32: 1009–1015.
3. Bolger, A. M., M. Lohse, and B. Usadel, 2014 Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30: 2114–2120.
4. Bosi, E., B. Donati, M. Galardini, S. Brunetti, M.-F. Sagot et al., 2015 MeDuSa: a multi-draft based scaffolder. Bioinformatics 31: 2443–2451.
5. Cantarel, B. L., P. M. Coutinho, C. Rancurel, T. Bernard, V. Lombard et al., 2009 The Carbohydrate-Active EnZymes database (CAZy): an expert resource for  Glycogenomics. Nucleic Acids Res. 37: D233-8.
6. Chen, S. H., and A. S. Yien Ting, 2015 Biodecolorization and biodegradation potential of recalcitrant triphenylmethane dyes by Coriolopsis sp. isolated from compost. J. Environ. Manage. 150: 274–280.
7. Consortium, T. U., 2019 UniProt: a worldwide hub of protein knowledge. Nucleic Acids Res. 47: D506–D515.
8. Gurevich, A., V. Saveliev, N. Vyahhi, and G. Tesler, 2013 QUAST: quality assessment tool for genome assemblies. Bioinformatics 29: 1072–1075.
9. Haghshenas, E., H. Asghari, J. Stoye, C. Chauve, and F. Hach, 2020 HASLR: Fast Hybrid Assembly of Long Reads. bioRxiv 2020.01.27.921817.
10. Mai, H., Y. Zhang, D. Li, H. C.-M. Leung, R. Luo et al., 2018 AC-DIAMOND v1: accelerating large-scale DNA–protein alignment. Bioinformatics 34: 3744–3746.
11. Page, A. J., C. A. Cummins, M. Hunt, V. K. Wong, S. Reuter et al., 2015 Roary: rapid large-scale prokaryote pan genome analysis. Bioinformatics 31: 3691–3693.
12. Sani, R. K., and U. C. Banerjee, 1999 Decolorization of triphenylmethane dyes and textile and dye-stuff effluent by Kurthia sp. Enzyme Microb. Technol. 24: 433–437.
13. Schwengers, O., T. Hain, T. Chakraborty, and A. Goesmann, 2019 ReferenceSeeker: rapid determination of appropriate reference genomes. bioRxiv 863621.
14. Seemann, T., 2014 Prokka: rapid prokaryotic genome annotation. Bioinformatics 30: 2068–2069.
15. Simão, F. A., R. M. Waterhouse, P. Ioannidis, E. V Kriventseva, and E. M. Zdobnov, 2015 BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs. Bioinformatics 31: 3210–3212.
16. Wick, R. R., L. M. Judd, C. L. Gorrie, and K. E. Holt, 2017 Unicycler: Resolving bacterial genome assemblies from short and long sequencing reads.(Research Article)(Report). PLoS Comput. Biol. 13: e1005595.
17. Zimin, A. V, G. Marçais, D. Puiu, M. Roberts, S. L. Salzberg et al., 2013 The MaSuRCA genome assembler. Bioinformatics 29: 2669–2677.


